cli/bench/types.go

```diff
@@ -8,3 +13,44 @@ type Results struct {
 	duration             float64 // Benchmark duration.
 	requestsPerSecond    int     // Cumber of requests per second - the main measured value.
 }
+
+// Interface for various types of requests
+type Request interface {
+	execute()
+}
+
+// InsertRequest data structure for insert request
+type InsertRequest struct {
+	ctx                 context.BenchCtx
+	tarantoolConnection *tarantool.Connection
+	results             *Results
+}
+
+// SelectRequest data structure for select request
+type SelectRequest struct {
+	ctx                   context.BenchCtx
+	tarantoolConnection   *tarantool.Connection
+	results               *Results
+	getRandomTupleCommand string
+}
+
+// UpdateRequest data structure for update request
+type UpdateRequest struct {
+	ctx                   context.BenchCtx
+	tarantoolConnection   *tarantool.Connection
+	results               *Results
+	getRandomTupleCommand string
+}
+
+// RequestsPool data structure for abstraction of a heap of identical requests
+type RequestsPool struct {
```

Comment:
It's not seems like some `pool`, it's more like a generator or something like that.

Reply:
Yes, I think the same